test/index.test.js

```diff
@@ -36,14 +36,38 @@ describe("WhatsAppAPI", function () {
         });
     });
 
+    describe("App secret", function () {
+        it("should work with empty as default", function () {
+            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN");
+            assert.equal(Whatsapp.appSecret, "");
+        });
+
+        it("should work with any specified app secret", function () {
+            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN", "YOUR_APP_SECRET");
+            assert.equal(Whatsapp.appSecret, "YOUR_APP_SECRET");
+        });
+    });
+
+    describe("Webhook verify token", function () {
+        it("should work with empty as default", function () {
+            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN");
+            assert.equal(Whatsapp.webhookVerifyToken, "");
+        });
+
+        it("should work with any specified webhook verify token", function () {
+            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN", "", "YOUR_WEBHOOK_VERIFY_TOKEN");
+            assert.equal(Whatsapp.webhookVerifyToken, "YOUR_WEBHOOK_VERIFY_TOKEN");
+        });
+    });
+
     describe("Version", function () {
         it("should work with v15.0 as default", function () {
             const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN");
             assert.equal(Whatsapp.v, "v15.0");
         });
 
         it("should work with any specified version", function () {
-            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN", "v13.0");
+            const Whatsapp = new WhatsAppAPI("YOUR_ACCESS_TOKEN", "", "", "v13.0");
```

Comment:
Just as a personal preference, I think it would look nicer if, rather than empty strings, they had a short explanation of what they are, just like the access token

Reply:
Yes, I agree. 

P.S. It's the disadvantage of positional arguments.